### PR TITLE
fix spellcheck errors

### PR DESCRIPTION
**Pull request by parbenc**

### CODE REVIEW

File: docs/Programmers_Guide.rst [hunk 1]
```diff
@@ -89,8 +89,8 @@ Test code for testing the scale contraction functionality and log metrics for F6
 Infrastructure
 ^^^^^^^^^^^^^^
 
-- CMake is used to build and package hipTensor. There are :code:`CMakeLists.txt` files throughout the code.
-- `Doxygen/Breathe/Sphinx/ReadtheDocs` are used to produce documentation. Content for the documentation is from:
+- CMake is used to build and package hipTensor. There are ``CMakeLists.txt`` files throughout the code.
```
Comment (neon60) [inline docs/Programmers_Guide.rst:92]: I am not sure, my changes here were valid. Please check in the built documentation.

Comment (parbenc) [reply docs/Programmers_Guide.rst:92]: This line is still valid, but it does nothing. I think it does no harm, but if there is a need for new commits I'll change it back to make the code more consistent.

**Comment by neon60** [comment]
@parbenc Please rebase this.

**Review by neon60 — DISMISSED** [review]

**Review by neon60 — APPROVED** [review]

**Review by saadrahim — APPROVED** [review]

**Review by LisaDelaney — APPROVED** [review]

**Comment by neon60** [comment]
[cgmillette](https://github.com/cgmillette) ready to merge in.

**Pull request merged**